features/us65_delete_borrower_personal_info.feature

```diff
@@ -0,0 +1,19 @@
+@us65_delete_borrower_personal_info
+
+Feature: Delete borrower personal information once the deed has been submitted
+  In order to ensure that Land Registry is not holding onto personal information
+  Borrowers personal details must be deleted from Land Registry records once the
+  mortgage application has been registered.
+
+Background:
+  Given I have valid deed data
+  And I create the deed via the Deed API
+  And I know the borrower id
+
+Scenario: Delete Single Borrowers Details
+  When I delete the deed via the Deed API
+  Then a status code of "200" is returned
```

Comment:
I'm not sure how can we do this but just thinking if it's enough to check that a 200 is returned or do we also want to check that the data has actually been deleted? Might want to double check with dev if this has been covered at unit test level, if so just checking the 200 will be ok.